en/docs/devops-and-ci-cd/configure-vpns-on-the-choreo-cloud-data-plane.md

```diff
@@ -0,0 +1,213 @@
+# Configure VPNs on the Choreo Cloud Data Plane
+
+Secure access to private networks from the Choreo cloud data plane is an essential use case for cloud data plane users. 
+
+Choreo allows this secure connection using [Tailscale](https://tailscale.com/). For this, Choreo provides a pre-installed Tailscale image that you can use in your Choreo project. This image acts as a forward proxy, allowing you to connect to your private endpoints through Tailscale's peer-to-peer [WireGuard](https://tailscale.com/kb/1035/wireguard) network tunnels.
```

Comment:
```suggestion
Choreo allows this secure connection using [Tailscale](https://tailscale.com/). For this, Choreo provides a pre-built image component with Tailscale paired with a forward proxy, which you can deploy in your Choreo project as a Service. Using this service, you can forward traffic to your external networks via Tailscale’s peer-to-peer [wireguard](https://tailscale.com/kb/1035/wireguard) network.
```

Reply:
Applied suggestion with a minor tweak to improve clarity.